### PR TITLE
Add early.radix.polydawn.io/hello-python

### PR DESCRIPTION
**Pull request by kkroening**

### CODE REVIEW

File: early.radix.polydawn.io/hello-python/module.tl [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+					"pypy3": "/pypy3"
+				},
+				"action": {
+					"exec": [
+						"/bin/bash", "-c",
+						"set -euo pipefail; echo $HOME; PYBIN=/pypy3/pypy3.5-6.0.0-linux_x86_64-portable/bin; VENV=$HOME/venv; $PYBIN/pypy3 $PYBIN/virtualenv-pypy $VENV; PYTHON=$VENV/bin/python3; $PYTHON -m ensurepip; $PYTHON -m pip wheel -w $HOME/wheels /src"
+					]
+				},
+				"outputs": {
+					"wheels": "/home/reuser/wheels"
```
Comment (kkroening) [inline early.radix.polydawn.io/hello-python/module.tl:22]: Better idea for where to put this? The hardcoded `reuser` is a bit sketch

File: early.radix.polydawn.io/hello-python/module.tl [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+	"imports": {
+		"root-debian": "catalog:early.polydawn.io/monolith/debian:v1.2017.01.04:linux-amd64",
+		"pypy3": "ingest:literal:tar:9RNHEQJMqihqhygqEf6rkpE5YaNNBAc4UDcXVg3wJKKS7Md2ZDp3ftSVkWPf6D8e2E",
```
Comment (kkroening) [inline early.radix.polydawn.io/hello-python/module.tl:4]: Gonna have to fix this import before merging, of course

File: early.radix.polydawn.io/hello-python/module.tl (hunk 1 above)
Comment (warpfork) [reply early.radix.polydawn.io/hello-python/module.tl:22]: I usually put things in dirs under `/task`, which is also guaranteed to be writable to you.  e.g. `/task/src`, `/task/out`, etc.  Looks like you could just as well set `VENV` to such an absolute path.

**Comment by warpfork** [comment]
I took these diffs, added a commit that moves them to live under an `early.drafts.*` name prefix, and merged thusly.

We'll still need to fix that janky literal, but I'm happy to have the files on master until we can get something better.

**Comment by warpfork** [comment]
Also, this is awesome and I'm glad you got the ball rolling on this constructive research! :tada:

**Pull request merged**